.vib/logstash/goss/goss.yaml

```diff
@@ -0,0 +1,36 @@
+http:
+  http://logstash:{{ .Vars.service.ports.http }}:
+    status: 200
+file:
+  {{ .Vars.persistence.mountPath }}:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: logstash
+  /bitnami/logstash/config/logstash.conf:
+    exists: true
+    filetype: symlink
+    mode: "0777"
+    owner: root
+    contains:
+    - /port.*{{ .Vars.containerPorts.http }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+    filetype: directory
+    mode: "3777"
+command:
+  {{- $rnd_address := printf "http://%s.com" (randAlpha 5) }}
+  test-pipeline:
+    # Logstash can take some time to parse and filter each input
+    exec: echo '98.83.179.51 - - [18/May/2011:19:35:08 -0700] "GET /css/main.css HTTP/1.1" 200 55 "{{ $rnd_address }}" "Mozilla/5.0"' >> /tmp/test_input && sleep 20 && cat /tmp/test_log
```

Comment:
If the IP `98.83.179.51` is "only to have one", I'd use another random one. I think is smarter.